docs/src/docs/versus.md

```diff
@@ -0,0 +1,88 @@
+
+Jupyter notebooks are a fixture in the data science community for good reason - they're an excellent tool for mixing code, text, graphs, and images in a single file as is often required for research and model development.
+
+Unfortunately, notebooks are not good at handling state or versioning. Although notebooks can be added to git, they're awkward to work with and resist the normal granular diff'ing that has made git so popular.
+
+Additionally, while you can run a model in a notebook, the model isn't durably serialized making it painful to share models with colleagues who don't use notebooks.
+
+#### ModelKits & Jupyter Notebooks
+
+We suggest continuing to use notebooks, but include a [Kitfile](./kitfile/kf-overview.md) in each, and at the end of the notebook execute a `kit pack` command to save the [serialized model, dataset, and code](./kitfile/kf-overview.md#example) from the notebook into a ModelKit for versioning, centralized tracking, and sharing. This allows the data science team to continue to use Jupyter Notebooks, but allows software engineering, product management, and infrastructure teams to easily run, track, and use the models outside of notebooks.
+
+### Containers
+
+We love containers - they're great for running and deploying models. But they're not a natural way to distribute or version code or datasets. It's common for data scientists to have limited experience writing dockerfiles or building containers so most organizations have a platform engineering or SRE team build the container for the model.
+
+#### ModelKits & Containers
+
+We suggest having data science and production operations teams discuss how a model will be deployed (in a container, as a side-car, etc...) early in the development cycle. If a container is going to be used, you can [include a dockerfile or container in a ModelKit](./kitfile/kf-overview.md#example) so that a model can be easily run locally or pushed through a standard deployment pipeline when needed.
+
+### Git
+
+Git is excellent at managing software projects that consist of a large number of small files. Unfortunately, git was _never designed to manage large binary objects_ like the serialized models and datasets that are critical to AI/ML projects. Although git LFS can be used, it doesn't work smoothly with git's own versioning workflows. Plus, many data scientists don't know or like working with git, increasing the likelihood of repo errors and friction between teams.
+
+#### ModelKits & Git
+
+Code related to model development is often easier to store in ModelKits where it is always in-sync with the Jupyter Notebook, serialized model, and datasets used during development. Larger codebases and code related to application integrations is best kept in git, but is often helpful to also package into the ModelKit ([a codebase can be cloned into a ModelKit](./kitfile/kf-overview.md#example) so that anyone can see the state of the code at the point that the project's ModelKit was versioned).
```

Comment:
Im not sure that `kitfile/kf-overview.md#example` exist. It wont fail but that hashtag / header is not real so it will just go to the top. You could use `./kitfile/kf-overview.html#kitfile-structure` which is the parent section or you can add a new header just for the example.

Reply:
Good catch, I'll fix that.